site/viamguides/src/smart-snack-dispenser/smart-snack-dispenser.md

```diff
@@ -0,0 +1,444 @@
+
+<!-- ### Watch the Video -->
+
+<!-- See a demonstration of the smart snack dispenser in action: -->
+
+<!-- <video id="fpK4J9NzD2Q"></video> -->
+
+<!-- ------------------------ -->
+## Set up your Raspberry Pi
+Duration: 5
+
+Before embedding the Raspberry Pi in the food dispenser, you will prepare it with an operating system.
+
+The Raspberry Pi boots from a USB flash drive (or microSD card). You need to install Raspberry Pi OS on a storage device that you will use with your Pi. For more details about alternative methods of setting up your Raspberry Pi, refer to the [Viam docs](https://docs.viam.com/installation/prepare/rpi-setup/#install-raspberry-pi-os).
+
+### Install Raspberry Pi OS
+
+1. Connect the USB flash drive (or microSD card) to your computer.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and launch it.
+   ![raspberry pi imager](assets/imager.webp)
+1. Click **CHOOSE DEVICE**. Select your model of Pi, which is Raspberry Pi 4.
+1. Click **CHOOSE OS**. Select **Raspberry Pi OS (64-bit)** from the menu.
+1. Click **CHOOSE STORAGE**. From the list of devices, select the storage device you intend to use in your Raspberry Pi.
+   ![raspberry pi storage](assets/osSelect.webp)
+1. Configure your Raspberry Pi for remote access. Click **Next**. When prompted to apply OS customization settings, select **EDIT SETTINGS**.
+1. Check **Set hostname** and enter the name you would like to access the Pi by in that field, for example, `test`.
+1. Select the checkbox for **Set username and password** and set a username (for example, your first name) that you will use to log into the Pi. If you skip this step, the default username will be `pi` (not recommended for security reasons). And specify a password.
+1. Connect your Pi to Wi-Fi so that you can run `viam-server` wirelessly. Check **Configure wireless LAN** and enter your wireless network credentials. SSID (short for Service Set Identifier) is your Wi-Fi network name, and password is the network password. Change the section `Wireless LAN country` to where your router is currently being operated.
+   ![raspberry pi hostname username and password](assets/generalSettings.webp)
+1. Select the **SERVICES** tab, check **Enable SSH**, and select **Use password authentication**.
+   ![raspberry pi enable SSH](assets/sshSettings.webp)
+   > aside negative
+   > Be sure that you remember the `hostname` and `username` you set, as you will need this when you SSH into your Pi.
+1. **Save** your updates, and confirm `YES` to apply OS customization settings. Confirm `YES` to erase data on the USB flash drive. You may also be prompted by your operating system to enter an administrator password. After granting permissions to the Imager, it will begin writing and then verifying the Linux installation to the USB flash drive.
+1. Remove the USB flash drive from your computer when the installation is complete.
```

Comment:
```suggestion
1. Remove the storage device from your computer when the installation is complete.
```
Accounts for use of USB or microSD card instead of having to spell out both.